test/e2e/storage/sanity.go

```diff
@@ -638,6 +652,12 @@ func (v volume) remove(vol *csi.Volume, volName string) {
 	framework.ExpectNoError(err, delete)
 }
 
+func skipIfNodeNotKnownWorker(nodeID string) {
```

Comment:
Instead of naming the function after how it is implemented, can you name it according to what it checks? We want to know whether that node can be restarted, so I think `canRestartNode` is better. It also matches the name `restartNode`, so it is obvious that the two are related.

Reply:
Did the requested change.